dlq-alert.tf

```diff
@@ -0,0 +1,30 @@
+// single alert to minify unnecessary cost because threshold used in here is minimal
+module "bulk-scan-exception-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "Bulk_Scan_DLQ_-_BSP"
+  alert_desc = "Triggers when bulk scan services record at least one dead lettered message within a 15 minutes window timeframe."
+
+  app_insights_query = <<EOF
+customEvents
+| where name == "DeadLetter"
```

Comment:
If this event name is case sensitive it's worth adding a comment wherever the event is logged

Reply:
There is a [comment](https://github.com/hmcts/bulk-scan-orchestrator/pull/346/files#diff-e2cea5982bf2edf9460ae03ca875e86fR179) and class has [static field](https://github.com/hmcts/bulk-scan-orchestrator/pull/346/files#diff-b3ce9c46ac8a322d7d7ff110b7f563c1R12)